src/Command/MigrateCommand.php

```diff
@@ -556,8 +566,16 @@ private function migrate(
                     $this->io->writeln('You did not set the <options=bold>--convert_tag_to_option</> option.');
                     $this->io->writeln('Choose this option if you want to convert your tags to an option field instead of a text field.');
 
-                    if (count($tags) > self::TAG_LIMIT) {
-                        $this->io->writeln(sprintf('More than %s tags was found in the asset file, you should not convert the tags.', self::TAG_LIMIT));
+                    if ($this->hasTooManyTagsToBeConvertedIntoMultiOptionAttributes($tags)) {
+                        $this->io->writeln(
+                            sprintf(
+                                'More than %s tags was found in the asset file, you should not convert the tags.',
```

Comment:
« You should not » or « it’s not possible »?

Reply:
done